docs/api/api_blueprint.apib

```diff
@@ -2005,7 +2010,8 @@ Authorized user must be admin.
             {
               "data": {
                 "attributes": {
-                  "name": "Travel & Outdoor"
+                  "name": "Travel & Outdoor",
+                  "system-image-url": "https://www.w3schools.com/html/pic_mountain.jpg"
```

Comment:
why would you put a w3schools image

Reply:
Its already being used in the project for sample image url https://github.com/fossasia/open-event-server/blob/development/app/factories/common.py#LC9

Reply:
yeah the problem as mario has stated many times, is that we cannot rely on external resources because they break